docker/package/tezos_voting_wizard.py

```diff
@@ -13,6 +13,15 @@
 import re
 
 from wizard_structure import *
+from dataclasses import dataclass
```

Comment:
IIRC we can't use `dataclass` because we still need to support Ubuntu 18.04, and that LTS has Python 3.6, whereas `dataclass` is introduced in 3.7 :cry: